src/main/java/com/pylon/spokestack/SpeechPipeline.java

```diff
@@ -243,6 +245,7 @@ public Builder() {
             this.config.put("sample-rate", DEFAULT_SAMPLE_RATE);
             this.config.put("frame-width", DEFAULT_FRAME_WIDTH);
             this.config.put("buffer-width", DEFAULT_BUFFER_WIDTH);
+            this.config.put("gain", DEFAULT_GAIN);
```

Comment:
This is not a good name for this configuration parameter. A parameter named "gain" would be a factor used to multiply the microphone's output. `automatic-gain` with a boolean value would make more sense.

Also, does it even make sense to configure this? We should not add configurables that can't be implemented across platforms.

Reply:
In both iOS and Android, it's unclear what effect AGC has on the VAD/ASR accuracy. AGC was not supported on the test platforms I was working with at the time. As with all things iOS, it's unclear whether AGC (which is enabled by default) can be disabled in the latest release. It could definitely be disabled in < iOS 11. 

I can rename, or I can remove the parameter.

Reply:
Let's pull it for now, so it's one less thing for our iOS guy to port.